components/Intro/index.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image'
+import banana from '../../public/images/bananaGuy.webp'
+import { useState } from 'react'
+import { useEffect } from 'react'
+
+const Intro = () => {
+  const [show, setShow] = useState(true)
+
+  const onClick = () => {
```

Comment:
Would recommend using more semantic names for the function such as "toggleVisibility" or "showBananaGuy" or "showCharacter" etc. rather than also naming it `onClick`

Reply:
Have changed the function name from onClick to hideBananaGuy